File: src/com/zxy/ijplugin/wechat_miniprogram/lang/wxml/WXMLLanguage.java

package com.zxy.ijplugin.wechat_miniprogram.lang.wxml;

import com.intellij.lang.xml.XMLLanguage;

public class WXMLLanguage extends XMLLanguage {

    public static String[] EVENT_ATTRIBUTE_PREFIX_ARRAY = new String[]{"bind", "catch", "bind:", "catch:", "mut-bind", "mut-bind:"};

    public static WXMLLanguage INSTANCE = new WXMLLanguage();

    private WXMLLanguage() {
        super(XMLLanguage.INSTANCE, "WXML");
    }

}

File: src/main/kotlin/com/zxy/ijplugin/wechat_miniprogram/lang/wxml/WXMLLanguage.kt
package com.zxy.ijplugin.wechat_miniprogram.lang.wxml

import com.intellij.lang.xml.XMLLanguage

object WXMLLanguage : XMLLanguage(INSTANCE, "WXML") {
    @JvmStatic
    var EVENT_ATTRIBUTE_PREFIX_ARRAY = arrayOf("bind", "catch", "bind:", "catch:", "mut-bind", "mut-bind:")

    @JvmStatic
    var INSTANCE = WXMLLanguage
}
